README.md

```diff
@@ -447,6 +447,8 @@ The following settings are optional and allow you to further configure your clus
 * `settings.kubernetes.image-gc-high-threshold-percent`: The percent of disk usage after which image garbage collection is always run.
 * `settings.kubernetes.image-gc-low-threshold-percent`: The percent of disk usage before which image garbage collection is never run.
 * `settings.kubernetes.provider-id`: This sets the unique ID of the instance that an external provider (i.e. cloudprovider) can use to identify a specific node.
+* `settings.kubernetes.kubelet-log-level`: Adjust the logging verbosity of the `kubelet` process.
+  The default log level is 2, with higher numbers enabling more verbose logging.
```

Comment:
Hindsight being what it is, `settings.kubernetes` would have been better named `settings.kubelet`.

The naming dates from very early in the project when it wasn't clear whether we'd want to ship control plane components some day; now it seems pretty certain that we won't.

With that in mind, I would prefer to treat `settings.kubernetes` and `settings.kubelet` as aliases - which some day they might be - and just call this `settings.kubernetes.log-level`.

Reply:
I was originally thinking we may want something like `settings.kubernetes.log-level.kubelet`, `settings.kubernetes.log-level.foo`, etc., thinking there might be other aspects of kubernetes as a whole that we might want to expose separate settings for. But I don't think that fits very cleanly either.

I will update to just be `log-level` and we can decide in the future if that is applicable to anything other than kubelet. 👍